### PR TITLE
[typescript react-query] remove unused pageParamKey on infinite queries

### PR DESCRIPTION
**Pull request by giubatt**
## Description

This parameter is not used anymore for infinite queries in react-query, which causes a typescript error when type checking if you have `noUnusedParameters` enabled. This PR removes that parameter from the function call, which is a breaking change in the API of any infinite query hooks

Related #174

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to not work as
      expected)
- [ ] This change requires a documentation update

## Screenshots/Sandbox (if appropriate/relevant):

Adding links to sandbox or providing screenshots can help us understand more about this PR and take
action on it as appropriate

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can
reproduce. Please also list any relevant details for your test configuration

- [x] ran `yarn test`, updating the codegen snapshots

**Test Environment**:

- OS: Linux Manjaro
- `@graphql-codegen/...`: latest
- NodeJS: 16

## Checklist:

- [x] I have followed the
      [CONTRIBUTING](https://github.com/the-guild-org/Stack/blob/master/CONTRIBUTING.md) doc and the
      style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules

## Further comments

Today if you use a genereted infinite query hook you're passing this string variable, anyone updating to a version using this change would have to remove that parameter from the hook call, but it's already unused anyway

### CODE REVIEW

**Review by saihaj — COMMENTED** [review]
this I think will be a breaking change?

**Comment by giubatt** [comment]
> this I think will be a breaking change?

It would yeah, anyone using infinite hooks would have to remove the first parameter from their hook call:
```diff
- const { data } = useSomeInfiniteQuery("key", { ...vars }, { ...options })
+ const { data } = useSomeInfiniteQuery({ ...vars }, { ...options })
```

File: .changeset/chatty-shrimps-greet.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+'@graphql-codegen/typescript-react-query': minor
```
Comment (saihaj) [inline .changeset/chatty-shrimps-greet.md:2]:
```suggestion
'@graphql-codegen/typescript-react-query': major
```

**Comment by saihaj** [comment]
looks like some tests need fixing here @giubatt

**Comment by giubatt** [comment]
> looks like some tests need fixing here @giubatt

updated the snapshots, should be working now

**Review by saihaj — APPROVED** [review]
thank you!

**Pull request merged**